src/main/java/uk/gov/pay/webhooks/healthcheck/HealthCheckResource.java

```diff
@@ -54,7 +59,13 @@ public HealthCheckResource(Environment environment) {
                     @ApiResponse(responseCode = "503", description = "Service unavailable. If any healthchecks fail")
             }
     )
-    public Set<Map.Entry<String, HealthCheck.Result>> healthCheck() {
-        return environment.healthChecks().runHealthChecks().entrySet();
+    public Response healthCheck() {
+        SortedMap<String, HealthCheck.Result> results = environment.healthChecks().runHealthChecks();
+
+        return Response.status(allHealthy(results.values()) ? OK : SERVICE_UNAVAILABLE).entity(results.entrySet()).build();
```

Comment:
Can we have a test for `503`?

Reply:
not easily, but can be done in a non-integration test (mock)

Reply:
we should be able to stop postgres/sqs containers as needed

Reply:
👍